src/main/java/module/specialEvents/Match.java

```diff
@@ -3,6 +3,7 @@
 import core.model.match.MatchEvent;
 import core.model.match.MatchType;
 import core.model.match.Weather;
+import org.jetbrains.annotations.Nullable;
```

Comment:
As discussed earlier, pointing this one out – we may need to make `annotations` an explicit dependency if we want to use this.

Reply:
yes I will do that

Reply:
done